Packs/CortexAttackSurfaceManagement/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.1.0",
+    "currentVersion": "2.0.0",
```

Comment:
Major version bumps usually mark breaking changes, or significant changes - is this the case here? If it's a feature and backwards compatible, I'd change it to a minor bump (1.1.0 -> 1.2.0)

Reply:
@dorschw , as mentioned in the previous comment there were some significant changes in the main playbook.  I don't really consider them "breaking" as the previous playbook was just setup as and example, and don't believe it was being used.  That being said, I don't mind using any version number that you recommend.  Let me know what change (if any) I should make, please.

Reply:
let's do `1.2.0`, just change the release note file name, and the version value under `pack_metadata.json`, and we'll be ready to merge it :)

Reply:
@dorschw , just pushed changes